Discopane UI (functional) tests are failing

Some time around this commit the Discopane functional tests started failing:
https://travis-ci.org/mozilla/addons-frontend/jobs/344970328

It doesn't seem to be related to a code change. I can reproduce it locally.

```
=================================== test session starts ===================================
platform darwin -- Python 2.7.13, pytest-3.2.3, py-1.5.2, pluggy-0.4.0
driver: Firefox
sensitiveurl: .* *** WARNING: sensitive url matches https://example.com:4000 ***
metadata: {'Platform': 'Darwin-16.7.0-x86_64-i386-64bit', 'Packages': {'pluggy': '0.4.0', 'py': '1.5.2', 'pytest': '3.2.3'}, 'Driver': 'Firefox', 'Capabilities': {}, 'Plugins': {'firefox': '0.1.1', 'metadata': '1.5.0', 'base-url': '1.4.1', 'variables': '1.7.0', 'xdist': '1.20.1', 'selenium': '1.11.1', 'forked': '0.2', 'html': '1.16.0', 'repeat': '0.4.1'}, 'Python': '2.7.13', 'Base URL': 'https://example.com:4000'}
baseurl: https://example.com:4000
rootdir: /Users/kumar/dev/addons-frontend/tests/ui, inifile: setup.cfg
plugins: xdist-1.20.1, variables-1.7.0, selenium-1.11.1, repeat-0.4.1, metadata-1.5.0, html-1.16.0, forked-0.2, firefox-0.1.1, base-url-1.4.1
collected 6 items

tests/ui/tests/test_discopane.py ..F.FF

======================================== FAILURES =========================================
______________________________ test_that_welcome_video_plays ______________________________

discovery_pane = <pages.discopane.DiscoveryPane object at 0x10a416250>

    @pytest.mark.nondestructive
    def test_that_welcome_video_plays(discovery_pane):
        """Test the welcome video."""
        assert not discovery_pane.is_close_video_displayed
        discovery_pane.play_video()
>       assert not discovery_pane.is_play_video_displayed
E       assert not True
E        +  where True = <pages.discopane.DiscoveryPane object at 0x10a416250>.is_play_video_displayed

tests/ui/tests/test_discopane.py:32: AssertionError
------------------------------------- pytest-selenium -------------------------------------
URL: https://example.com:4000/en-US/firefox/discovery/pane/58.0/darwin/normal#play
Driver log: /private/var/folders/8g/npvxp6j96sn_07b1f6b3h0qm0000gn/T/pytest-of-kumar/pytest-0/test_that_welcome_video_plays0/driver.log
WARNING: Failed to gather log types: Message: GET /session/54d5b585-30ec-4848-801e-fac86bd0bac6/log/types did not match a known command
___________________________________ test_addon_installs ___________________________________

discovery_pane = <pages.discopane.DiscoveryPane object at 0x10a416c10>
firefox = <foxpuppet.foxpuppet.FoxPuppet object at 0x10a2c0b90>
notifications = <function notifications at 0x10a1d5488>

    @pytest.mark.nondestructive
    def test_addon_installs(discovery_pane, firefox, notifications):
        """Test addon installation from discovery pane."""
        addon = discovery_pane.addons[0]
        addon.install()
        firefox.browser.wait_for_notification(
>           notifications.AddOnInstallConfirmation).install()

tests/ui/tests/test_discopane.py:50:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
.tox/discopane-ui-tests/lib/python2.7/site-packages/foxpuppet/windows/browser/window.py:63: in wait_for_notification
    message=message)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <selenium.webdriver.support.wait.WebDriverWait (session="9646760f-5260-4a40-8419-4eb1b4cc2c74")>
method = <function <lambda> at 0x10a42ef50>
message = 'AddOnInstallConfirmation was not shown.'

    def until(self, method, message=''):
        """Calls the method provided with the driver as an argument until the \
            return value is not False."""
        screen = None
        stacktrace = None

        end_time = time.time() + self._timeout
        while True:
            try:
                value = method(self._driver)
                if value:
                    return value
            except self._ignored_exceptions as exc:
                screen = getattr(exc, 'screen', None)
                stacktrace = getattr(exc, 'stacktrace', None)
            time.sleep(self._poll)
            if time.time() > end_time:
                break
>       raise TimeoutException(message, screen, stacktrace)
E       TimeoutException: Message: AddOnInstallConfirmation was not shown.

.tox/discopane-ui-tests/lib/python2.7/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException
------------------------------------- pytest-selenium -------------------------------------
URL: https://example.com:4000/en-US/firefox/discovery/pane/58.0/darwin/normal
Driver log: /private/var/folders/8g/npvxp6j96sn_07b1f6b3h0qm0000gn/T/pytest-of-kumar/pytest-0/test_addon_installs0/driver.log
WARNING: Failed to gather log types: Message: GET /session/9646760f-5260-4a40-8419-4eb1b4cc2c74/log/types did not match a known command
___________________________________ test_theme_installs ___________________________________

discovery_pane = <pages.discopane.DiscoveryPane object at 0x10a4199d0>
firefox = <foxpuppet.foxpuppet.FoxPuppet object at 0x10a3ef6d0>
notifications = <function notifications at 0x10a1d5488>

    @pytest.mark.nondestructive
    def test_theme_installs(discovery_pane, firefox, notifications):
        """Test theme install from discovery pane."""
        theme = discovery_pane.themes[0]
        theme.install()
        firefox.browser.wait_for_notification(
>           notifications.AddOnInstallConfirmation).install()

tests/ui/tests/test_discopane.py:62:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
.tox/discopane-ui-tests/lib/python2.7/site-packages/foxpuppet/windows/browser/window.py:63: in wait_for_notification
    message=message)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <selenium.webdriver.support.wait.WebDriverWait (session="4da0b083-97e3-4d4c-972e-e49c6f16bdfc")>
method = <function <lambda> at 0x10a4ae758>
message = 'AddOnInstallConfirmation was not shown.'

    def until(self, method, message=''):
        """Calls the method provided with the driver as an argument until the \
            return value is not False."""
        screen = None
        stacktrace = None

        end_time = time.time() + self._timeout
        while True:
            try:
                value = method(self._driver)
                if value:
                    return value
            except self._ignored_exceptions as exc:
                screen = getattr(exc, 'screen', None)
                stacktrace = getattr(exc, 'stacktrace', None)
            time.sleep(self._poll)
            if time.time() > end_time:
                break
>       raise TimeoutException(message, screen, stacktrace)
E       TimeoutException: Message: AddOnInstallConfirmation was not shown.

.tox/discopane-ui-tests/lib/python2.7/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException
------------------------------------- pytest-selenium -------------------------------------
URL: https://example.com:4000/en-US/firefox/discovery/pane/58.0/darwin/normal
Driver log: /private/var/folders/8g/npvxp6j96sn_07b1f6b3h0qm0000gn/T/pytest-of-kumar/pytest-0/test_theme_installs0/driver.log
WARNING: Failed to gather log types: Message: GET /session/4da0b083-97e3-4d4c-972e-e49c6f16bdfc/log/types did not match a known command
=========================== 3 failed, 3 passed in 63.29 seconds ===========================
ERROR: InvocationError: '/Users/kumar/dev/addons-frontend/.tox/discopane-ui-tests/bin/pytest --driver Firefox tests/ui/tests/test_discopane.py'
```

--- a/tests/ui/conftest.py
+++ b/tests/ui/conftest.py
@@ -50,4 +50,5 @@ def firefox_options(firefox_options):
     firefox_options.set_preference(
         'extensions.install.requireBuiltInCerts', False)
     firefox_options.add_argument('-foreground')
+    firefox_options.set_preference('ui.popup.disable_autohide', True)
     return firefox_options

